TrackingTools/GsfTracking/python/CkfElectronCandidateMaker_cff.py

```diff
@@ -40,7 +40,8 @@
 TrajectoryBuilderForElectrons.TTRHBuilder = 'WithTrackAngle'
 TrajectoryBuilderForElectrons.updator = 'KFUpdator'
 
-
+from Configuration.ProcessModifiers.seedingDeepCore_cff import seedingDeepCore
+seedingDeepCore.toModify(TrajectoryBuilderForElectrons, maxPtForLooperReconstruction = cms.double(0.0) )
```

Comment:
`maxPtForLooperReconstruction = 0.0`

Reply:
mmh, this is not possible. TrajectoryBuilderForElectrons has no attribute called maxPtForLooperReconstruction [1] and this is used everywhere in CMS [2], also from [3]. The only similar variable is this one [4], but is not accessible (and is actually different). 

I am doing something wrong? I really do not uderstand, is there any parent class that I'm missing? 

[1]https://cmssdt.cern.ch/dxr/CMSSW/source/RecoTracker/CkfPattern/python/CkfTrajectoryBuilder_cfi.py
[2] https://cmssdt.cern.ch/dxr/CMSSW/search?q=maxPtForLooperReconstruction&case=true
https://cmssdt.cern.ch/dxr/CMSSW/source/RecoTracker/CkfPattern/plugins/GroupedCkfTrajectoryBuilder.h#195
[3] https://cmssdt.cern.ch/dxr/CMSSW/source/RecoTracker/CkfPattern/python/GroupedCkfTrajectoryBuilder_cfi.py
[4]https://cmssdt.cern.ch/dxr/CMSSW/source/RecoTracker/CkfPattern/plugins/GroupedCkfTrajectoryBuilder.h#195

Reply:
ok you're right, sorry for the noise. it looked like you are modifying an existing parameter, it's fine as it is.

Reply:
ok good, np :)